identity-aware-proxy/IAPClient/README.md

```diff
@@ -3,79 +3,23 @@
 This sample shows how to authenticate an http request when requesting a web page from
 a Google App Engine application protected by the [Google Cloud Identity-Aware Proxy](http://cloud.google.com/iap).
 
-This sample requires [.NET Core 2.0](
+This sample requires [.NET Core 3.1](
     https://www.microsoft.com/net/core) or later.  That means using
-[Visual Studio 2017](
+[Visual Studio 2019](
     https://www.visualstudio.com/), or the command line.
 
 ## Build and Run
 
 1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/dotnet/docs/setup).**
 
-2.  Follow the instructions here
+1.  Follow the instructions here

```

Comment:
I think these links would be better with the text that follows them as the link, and no "follow the instructions here". So:

1. [Create an App Engine application and enable Identity-Aware Proxy (IAP)](https://cloud.google.com/iap/docs/authentication-howto#iap-make-request-csharp)
1. [Download service account credentials and record your IAP client ID](https://cloud.google.com/iap/docs/authentication-howto#iap-make-request-csharp)